store/storetest/oauth_store.go

```diff
@@ -411,12 +410,14 @@ func testOAuthStoreDeleteApp(t *testing.T, ss store.Store) {
 		t.Fatal(err)
 	}
 
-	s1 := model.Session{}
+	s1 := &model.Session{}
 	s1.UserId = model.NewId()
 	s1.Token = model.NewId()
 	s1.IsOAuth = true
 
-	store.Must(ss.Session().Save(&s1))
+	s1, err := ss.Session().Save(s1)
+
```

Comment:
nit: this blank line between the `Save` and the `require.Nil` looks odd to me. For me is clearly the same "thing". Save and verify that it have been saved properly. Not a stopper but would be nice to remove this blank line from everywhere you have inserted it.